### ISSUE TITLE
Bug: max-len ignore block comments

### ISSUE THREAD

**Issue by Ranguna** [eslint/eslint#15543]
### What rule do you want to change?

max-len

### What change to do you want to make?

Generate fewer warnings

### How do you think the change should be implemented?

A new option

### Example code

```js
exists
	? topic
	: /* istanbul ignore next: this should never happen, but let's keep it just in case */ topic.create().then(/* istanbul ignore next */ ([newTopic]) => newTopic)
```

### What does the rule currently do for this code?

With the following:
```
"max-len": [
	"error",
	{
		"code": 150,
		"ignoreComments": true
	}
]
```

We get this: This line has a length of 163. Maximum allowed is 150.

### What will the rule do after it's changed?

The existing `ignoreComments` option in `max-len` only applies to trailing comments or comments on their own line.

It would be cool to have an option to ignore all block comments.

### Participation

- [ ] I am willing to submit a pull request to implement this change.

**Comment by nzakas** [comment]
Thanks for reaching out. This rule is working as intended. We can’t know if any given comment in the middle of a statement has meaning or not, so we don’t warn on that. 

Additionally, we are no longer making updates to rules that just enforce stylistic preferences, as this rule does. If this behavior is important to you, you can always create a custom rule by copying the existing one and modifying it.

**Comment by Ranguna** [comment]
Ah this was actually an enhancement not a bug report.

I tried to open this as a bug, but changed it to a change request, looks like github kept the bug prefex.
